src/fluree/db/indexer/default.cljc

```diff
@@ -402,7 +402,7 @@
         novel?     (fn [node]
                      (or (seq remove-preds)
                          (seq (index/novelty-subrange node t novelty))))]
-    (->> (index/tree-chan conn root novel? (constantly true) 1 refresh-xf error-ch)
+    (->> (index/tree-chan conn root novel? 4 refresh-xf error-ch)
```

Comment:
`4` feels kind of magic-number-y here in a way that `1` doesn't as much. Should it be def'd somewhere to give it a name?

Reply:
It is. I just felt like we should be doing this with higher parallelization, and 4 seemed like a fine number. In reality though, this type of thing should be parameterized. I'll change it back to one and make a note to parameterize this later.